instagram_private_api/endpoints/misc.py

```diff
@@ -86,6 +86,19 @@ def direct_v2_inbox(self):
         """Get v2 inbox"""
         return self._call_api('direct_v2/inbox/')
 
+    def direct_v2_thread(self, thread, **kwargs):
```

Comment:
Please following the current library convention and rename the parameter to ``thread_id``